frontend/src/app/helpers/default.error-matcher.ts

```diff
@@ -0,0 +1,11 @@
+import { FormControl, FormGroupDirective, NgForm } from "@angular/forms";
+import { ErrorStateMatcher } from "@angular/material/core";
+
+export class MyErrorStateMatcher implements ErrorStateMatcher {
```

Comment:
Could we change this name as well? `PasswordStateMatcher`?

Reply:
As per another comment, error state matcher can be used in other fields, so it's best to keep the name general